code/modules/reagents/reagent_containers/jug.dm

```diff
@@ -0,0 +1,54 @@
+//Not to be confused with /obj/item/reagent_containers/food/drinks/bottle
+
+
+/obj/item/reagent_containers/glass/jug
+	name = "Jug"
```

Comment:
I believe the default for things is lowercase name. Since it's not a real name but just an object